content.php

```diff
@@ -10,40 +10,56 @@
     $imageAttributes = wp_get_attachment_image_src( get_post_thumbnail_id(), 'full' ); // returns an array
     
     if ( $imageAttributes ) {
-        $articleStyle = 'style="background-image: url( ' . $imageAttributes[0] . ' )"';
+        $articleStyle = 'style="background-image: background: linear-gradient(45deg, rgba(0,0,0,0.3) 0%,rgba(0,0,0,0.1) 48%,rgba(0,0,0,0) 100%), url( ' . $imageAttributes[0] . ' )"';
```

Comment:
`$articleStyle` isn't used anymore, remove this (double check if it really isn't used anymore)